Tools/Pipeline/Windows/MainView.cs

```diff
@@ -160,16 +161,11 @@ private void OnPropertyGridPropertyValueChanged(object s, PropertyValueChangedEv
             if (args.ChangedItem.Label == "References")
                 _controller.OnReferencesModified();
 
-            // TODO: This is the multi-select case which needs to be handled somehow to support undo.
-            if (args.OldValue == null)
-                return;
-
-            var obj = _propertyGrid.SelectedObject;
+            var obj = _oldValues.Count == 1 ? _propertyGrid.SelectedObject : new ContentItem();
```

Comment:
This could be written so that a `new ContentItem` we don't actually use is not being allocated.